src/python/devcontainer-feature.json

```diff
@@ -73,15 +73,19 @@
     "vscode": {
       "extensions": [
         "ms-python.python",
-        "ms-python.vscode-pylance"
+        "ms-python.vscode-pylance",
+        "ms-python.autopep8"
```

Comment:
This sounds good to me, we always supported it when it was part of `ms-python.python`. So these changes make sense to me. 

// cc @bamurtaugh unless there are any concerns.